### ISSUE TITLE
"model.classify" is not capturing training picks

### ISSUE THREAD

**Issue by javak87** [seisbench/seisbench#147]
Dear Developers,
I'm using seisbench to transfer learning of phasenet.
After training, the f1score for P pick increased to 0.92, and for S pick it raised to 0.86.
I check all steps of training and it's ok, but when I applied "model.classify", the performance of p picks and s picks decreased less than the phasenet model(without training).
I don't know how "classify" works. Because we are feeding the stream to "classify", this method should select the sliding window of 30 seconds and feed it into phasenet for classification. However, when I applied this method, the training picks is not captured (it should identify at least 92% of p picks of training picks).
could you please tell me how can I overcome this inconsistency?
Best regards,
Javad

**Comment by yetinam** [comment]
Hey @javak87 ,

I suspect the issue is related to the thresholds that the model uses to define a pick. Try lowering the thresholds. You can do this by passing `P_threshold` and `S_threshold` to `classify`. For example, `model.classify(stream, P_threshold=0.1, S_threshold=0.1)`. In addition, I'd suggest manually inspecting some probability curve output by `annotate`. This might help to debug the issue.

Please let me know if this solves your issue.

**Comment by yetinam** [comment]
Hey @javak87,
could you a quick heads-up if your issue is solved?

**Comment by javak87** [comment]
Dear @yetinam 
Thank you for your response.
Unfortunately, I couldn't find the error, and this problem is still present.
I will try to make sure about the workflow for training and after that I let you know.
Thanks

**Comment by javak87** [comment]
Dear @yetinam,
I think you should close this issue. When I used new data and try to apply transfer learning for EQT, the loss behavior was normal, but the the result got worse. I though that maybe this issue belong the classify, but I think the problem might be related to loss function, or data augmentation.
Thanks
